sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/TestEnvironment.cs

```diff
@@ -16,6 +16,9 @@ public static class TestEnvironment
         /// <summary>The name of the environment variable from which the Form Recognizer resource's API key will be extracted for the live tests.</summary>
         public const string ApiKeyEnvironmentVariableName = "FORM_RECOGNIZER_API_KEY";
 
+        /// <summary>The name of the environment variable from which the Form Recognizer resource's API key will be extracted for the live tests.</summary>
+        public const string BlobContainerSASUrlEnvironmentVariableName = "FORM_RECOGNIZER_BLOB_CONTAINER_SAS_URL";
```

Comment:
nit: the doc description of the field has not been updated.

Also, I remember being told that acronyms in variable names in .NET should follow camel case (`Sas` instead of `SAS`). Not sure if just a matter of preference, but it seems a pattern we've been following in other places (like in `ContentType.Jpeg`, `ContentType.Pdf`).